runtime/server/controller.go

```diff
@@ -61,17 +61,21 @@ func (s *Server) ListResources(ctx context.Context, req *runtimev1.ListResources
 		return strings.Compare(an.Name, bn.Name)
 	})
 
+	// Check if the requester is an admin
+	claims := auth.GetClaims(ctx)
+	isAdmin := claims.SecurityClaims().Admin()
```

Comment:
The problem here is that `ListResources` is used many places, e.g. both the main dashboard listings and the status page listings. Outside the status page listings, people *do* expect the policies to be enforced, even for admins.

One option is to add a `skip_checks` bool flag to the `ListResources` and `GetResource` RPCs that the UI can pass in the contexts where it wants the checks to be skipped (and of course, setting it to true for a non-admin should error).